requirements/edx/github.in

```diff
@@ -74,3 +74,5 @@ git+https://github.com/edx/django-ratelimit-backend.git@v2.0.1a5#egg=django-rate
 
 git+https://github.com/open-craft/xblock-poll@922cd36fb1c3cfe00b4ce03b19a13185d136447d#egg=xblock-poll==1.10.2
 git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.3.4#egg=xblock-drag-and-drop-v2==2.3.4
+# FIXME: This is just to allow tests to run
+git+https://github.com/open-craft/xblock-lti-consumer@kshitij/bb-3899/pii-flag#egg=lti-consumer-xblock==2.9.0
```

Comment:
so you are intended to remove this after the merge of `xblock-lti-consumer` PR?

Reply:
Yes, I will revert this change, and bump up the version instead. Without using the latest xblock version, all tests fail since a required import is absent.